<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DevSettingBase_Xover</class>
 <widget class="QDialog" name="DevSettingBase_Xover">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="head">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="bandpass" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>281</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
     border: 1px solid #8f8f91;
     border-radius: 4px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }
QPushButton:checked{
	background-color: rgb(0, 178, 0);
}</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Low Pass</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>High Pass</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Freq</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Stages</string>
    </property>
   </widget>
   <widget class="QComboBox" name="low_stages">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>S_12dB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S_24dB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S_36dB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S_48dB</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="low_freq">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>20</number>
    </property>
    <property name="maximum">
     <number>20000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="high_freq">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>20</number>
    </property>
    <property name="maximum">
     <number>20000</number>
    </property>
   </widget>
   <widget class="QComboBox" name="high_stages">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>S_12dB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S_24dB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S_36dB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S_48dB</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="bypass">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Bypass</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
